gensim/models/word2vec.py

```diff
@@ -909,6 +913,879 @@ def accuracy(self, questions, restrict_vocab=30000, most_similar=most_similar):
         return sections
 
 
+    def argTopNSorted_nda(self, inArray_nda, topn=1):
+        """
+        Returns an array containing the indexes to produce the topn largest
+        items in sorted order. Runs in n*log(k) time where n is inArray_nda
+        size and k is topn.
+
+        inArray_nda must contain numbers.
+        """
+        if topn == 1:
+            return argmax(inArray_nda)
+        unsortedTopNIdxs_nda = argpartition(-inArray_nda, topn)[:topn]
+        subIdxs_nda = argsort(-inArray_nda[unsortedTopNIdxs_nda])
+        sortedTopNIdxs_nda = unsortedTopNIdxs_nda[subIdxs_nda]
+        return sortedTopNIdxs_nda
+
+
+    def argTopNSorted_2D_nda(self, inArray_nda, topn=1, axis=0):
```

Comment:
Python uses `underscores_for_identifiers` by convention, not `camelCase` ([PEP 8](https://www.python.org/dev/peps/pep-0008/)).